modules/dnn/src/layers/fast_convolution/winograd_3x3s1_f63.cpp

```diff
@@ -1093,59 +1092,63 @@ int runWinograd63(InputArray _input, OutputArray _output, const Ptr<FastConv2d>&
 #else // ARMv7 16 registers.
 
                         // init 16 registers. FMA/load ratio = 32/12
-                        float32x4_t r00 = vdupq_n_f32(0.0f), r01 = r00, r02 = r00, r03 = r00;
-                        float32x4_t r04 = r00, r05 = r00, r06 = r00, r07 = r00;
-                        float32x4_t r08 = r00, r09 = r00, r10 = r00, r11 = r00;
-                        float32x4_t r12 = r00, r13 = r00, r14 = r00, r15 = r00;
+                        float32x2_t q00 = vdup_n_f32(0.0f), q01 = q00, q02 = q00, q03 = q00,
```

Comment:
could you please explain why you use halves of NEON registers on ARMv7? ARMv7 still has 128-bit NEON registers, I don't see why not use all of them

Reply:
Thanks for code reviewing. As @nihui's commented, `vmlaq_lane_f32 ` is the best substitute for `vfmaq_laneq_f32 ` under the ARMv7 platform.
Another option is `vmlaq_n_f32`, it will be parsed into two arm assembly code `fmul.4s             v24, v22, v20[0]` and `fadd.4s v3, v3, v24`. And `vmlaq_lane_f32` will be parsed into only one arm assembly code `vmla.f32 q8, q5, d0[1]` on ARMv7. 
At the same time, two consecutive half-length register loads will be converted into a 128bit load during loading data to register, so the data load time is the same.